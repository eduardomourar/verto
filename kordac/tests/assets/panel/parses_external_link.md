{panel type="extra-for-experts" summary="ExternalLink"}
The formal term for working out the cost of an algorithm is {external-link url="https://en.wikipedia.org/wiki/Analysis_of_algorithms" text="algorithm analysis"},
and we often refer to the cost as the algorithm's *complexity*.
{panel end}
