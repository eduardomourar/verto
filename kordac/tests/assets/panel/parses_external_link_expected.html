<div class="clearfix">
<ul class="collapsible panel" data-collapsible="accordion">
<li class="panel-selector panel-extra-for-experts">
<div class="collapsible-header">
<div class="panel-heading">Extra For Experts: ExternalLink</div>
<div class="dropdown-menu-arrow">â–¼</div>
</div>
<div class="collapsible-body">
<p>The formal term for working out the cost of an algorithm is <a href="https://en.wikipedia.org/wiki/Analysis_of_algorithms">algorithm analysis</a>,
and we often refer to the cost as the algorithm's <em>complexity</em>.
</p>
</div>
</li>
</ul>
</div>